src/status_im/ui/components/action_button/styles.cljs

```diff
@@ -2,11 +2,11 @@
   (:require-macros [status-im.utils.styles :refer [defstyle defnstyle]])
   (:require [status-im.utils.platform :as p]
             [status-im.ui.components.styles :refer [color-white
-                                                 color-light-blue-transparent
-                                                 color-light-blue
-                                                 color-light-gray
-                                                 color-black
-                                                 color-gray4]]))
+                                                    color-light-blue-transparent
+                                                    color-light-blue
+                                                    color-light-gray
+                                                    color-black
+                                                    color-gray4]]))
```

Comment:
`:refer` -> `:as`, p -> platform, etc

Reply:
no please, no, PR is huge, and there were no changes in this file

Reply:
#3141